lstmcpipe/config/pipeline_config.py

```diff
@@ -207,14 +207,21 @@ def parse_config_and_handle_global_vars(loaded_config):
         f"source {loaded_config['source_environment']['source_file']}; "
         f"conda activate {loaded_config['source_environment']['conda_env']}; "
     )
-    config["source_environment"] = src_env
+    # 2.1 - Parse slurm user config account
+    if config["slurm_config"]:
```

Comment:
This will never be true.
Do you mean 
```python
if loaded_config.get("slurm_config")
```
?

Reply:
you're completely right @LukasNickel ! 
it was `loaded_config` indeed (thanks tests too... yes, pending thing to do...)

Reply:
Use `.get()` if this is optional in the config. If you want to enforce it, adapt the test (Which should probably be done anyway)

Reply:
Thanks @LukasNickel ;-)